music_assistant/providers/lastfm_scrobble/__init__.py

```diff
@@ -32,8 +32,9 @@ async def setup(
     """Initialize provider(instance) with given configuration."""
     provider = LastFMScrobbleProvider(mass, manifest, config)
     pylast.logger.setLevel(provider.logger.level)
-    if provider.logger.level == logging.DEBUG:
-        # httpcore is quite spammy without providing useful information 99% of the time
+
+    # httpcore is very spammy on debug without providing useful information 99% of the time
+    if logging.getLogger("httpcore").level <= logging.DEBUG:
         logging.getLogger("httpcore").setLevel(logging.INFO)
```

Comment:
```suggestion
    if provider.logger.level == logging.DEBUG:
        logging.getLogger("httpcore").setLevel(logging.INFO)
    else:
        logging.getLogger("httpcore").setLevel(logging.WARNING)
```

These loggers are super verbose - just silence them and only raise to info if your provider has debug logging enabled. Remember that you need to specify the log level, not the other way around.

Reply:
Yeah that makes a lot of sense. Although I'm still wondering if it's something we need to tackle in the provider, I can't imagine pylast is the only thing that uses httpcore/httpx?